src/quo2/components/reactions/reaction.cljs

```diff
@@ -12,31 +12,32 @@
   [rn/touchable-opacity
```

Comment:
Nit: the name space of this component is wrong - it's not to do with your changes but perhaps while you are working on these files you can migrate it to the correct location? 
I.e these should be in .selectors. To match figma and the name should match that of figma.

Reply:
Additionally these files should only export one component and it seems like this file has two in it, can we adjust that and map with fi hmm a 1-1 as we have in the rest of our components? Makes things easier to maintain and to understand what we have in place etc 🙂